composer.json

```diff
@@ -95,7 +100,8 @@
             "library/translation.inc.php"
         ],
         "psr-4" : {
-            "OpenEMR\\" : "src"
+            "OpenEMR\\" : "src",
+            "Modules\\" : "modules"
```

Comment:
hi @adunsulag , Was hoping to get your input since you've put a lot of thought into modules and modular coding in OpenEMR. The above will essentially march through the modules directory where there is class code combined with other code. Is that ok or is there another way we should be thinking about in organizing code (and classes) in modules?

Reply:
This is no longer a module(really plugin) it just resides in modules directory. I can change to Modules/oeFax.